.github/workflows/deploy-dev-testnet.yml

```diff
@@ -30,7 +30,7 @@ jobs:
 
       - name: Install bos CLI
         run: |
-          curl --proto '=https' --tlsv1.2 -LsSf https://github.com/FroVolod/bos-cli-rs/releases/download/v0.3.2/bos-cli-installer.sh | sh
+          curl --proto '=https' --tlsv1.2 -LsSf https://github.com/bos-cli-rs/bos-cli-rs/releases/download/v0.3.15/bos-cli-installer.sh | sh
```

Comment:
Can we use `latest` instead of a fixed version?

`curl --proto '=https' --tlsv1.2 -LsSf https://github.com/bos-cli-rs/bos-cli-rs/releases/latest/download/bos-cli-installer.sh | sh`

Reply:
oh yes, but I m concerned about one thing, if anything breaks in latest, it would effect us as well, and also I don't think the versions update that quickly. 
wdyt?

Reply:
I was thinking the other way, that if something is not working in the current version, we don't have to make changes in the DevHUB repo when the fix version is published. Of course we need then to trust that there is good regression test coverage then.. Maybe it's too risky?

I'll resolve this. We can go with 0.3.15 for now 👍